components/higher-order/with-focus-return/index.js

```diff
@@ -23,7 +23,7 @@ function withFocusReturn( WrappedComponent ) {
 			if (
 				this.activeElement && (
 					( document.activeElement && wrapper && wrapper.contains( document.activeElement ) ) ||
-					! document.activeElement
+					( ! document.activeElement || document.body === document.activeElement )
```

Comment:
Yeah, there might be some possibility of this existing somehow, at one point JSDOM would not handle this to the HTML spec, so `! document.activeElement` would be falsy.  I think keeping it in is okay.  Adds to the readability a bit in my opinion as well.